server/db/connect.js

```diff
@@ -0,0 +1,17 @@
+const { MongoClient } = require("mongodb");
+
+const dbUrl = "mongodb://localhost:27017/fib";
+const client = new MongoClient(dbUrl);
+
```

Comment:
What if mongo is not installed on my machine and running locally? Maybe create a remote instance and provide the user a connection to it.